src/smart-components/Channel/components/MessageList/index.tsx

```diff
@@ -156,7 +156,7 @@ const MessageList: React.FC<MessageListProps> = ({
             return (
               <MessageProvider message={m} key={m?.messageId} isByMe={isByMe}>
                 <Message
-                  handleScroll={handleScroll}
```

Comment:
I want to remove this prop ~ we need to probably move `handleMessageHeightChange` to ChannelProvider

Reply:
or, better make messageList a UI component 
~brainstorming here

Reply:
Yeah
> move handleMessageHeightChange to ChannelProvider
This is a way that I was also considering